### PR TITLE
feat: add benchmark document array constructor

### PR DESCRIPTION
**Pull request by bwanglzu**

### CODE REVIEW

File: src/document_array_construct.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import os
+import uuid
+
+from faker import Faker
```
Comment (JoanFM) [inline src/document_array_construct.py:4]: remember to add to requirements

File: src/document_array_construct.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+from jina import Document, DocumentArray, __version__
+from jina.types.arrays.memmap import DocumentArrayMemmap
+from jina.logging.profile import TimeContext
```
Comment (JoanFM) [inline src/document_array_construct.py:8]: this is annoying to use, too verbose. I created a different TimeContext instance without the noise

Comment (bwanglzu) [reply src/document_array_construct.py:9]: okay

**Comment by maateen** [comment]
LGTM

**Review by JoanFM — APPROVED** [review]

**Pull request merged**